<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>530</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to cart</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>90</y>
      <width>331</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QLabel" name="totalLabel">
       <property name="text">
        <string>Total: $0.00</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="removeButton">
       <property name="text">
        <string>Remove from cart</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QComboBox" name="discountComboBox">
       <property name="currentText">
        <string>No discount</string>
       </property>
       <item>
        <property name="text">
         <string>No discount</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>%10 discount</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="giftWrapping">
       <property name="text">
        <string>Gift Wrapping</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="applyDiscountButton">
       <property name="text">
        <string>Apply Discount</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="confirmOrder">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>430</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm Order</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="cartNotification">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>520</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="discountNotification">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>260</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="cartListWidget">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>361</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="productImageLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>400</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="productListWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>472</width>
      <height>291</height>
     </rect>
    </property>
    <property name="modelColumn">
     <number>0</number>
    </property>
   </widget>
   <widget class="QTextEdit" name="productDetailsTextEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>472</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>472</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>472</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
